includes/utils.inc

```diff
@@ -1232,6 +1235,12 @@ function wf_crm_get_fields($var = 'fields') {
         'name' => t('End Date'),
         'type' => 'date',
       );
+      $fields['membership_auto_renew'] = array(
+        'name' => t('Auto-renew Membership?'),
+        'type' => 'select',
+        'expose_list' => TRUE,
+        'value' => 0,
+      );
```

Comment:
This is slightly different from how CiviCRM core works, but I don't have a problem with an extra parameter - we need to consider the upgrade impact though as I think this changes previous default behaviour?

Reply:
> we need to consider the upgrade impact though as I think this changes previous default behaviour?

good point, will check that.

Reply:
>>> This is slightly different from how CiviCRM core works,

BTW this is how civicrm core works actually, if you are configuring a membership signup/renewal "contribution page" then you can either force the membership to be auto-renewed by default of give the option to the user 

![2019-11-04 18_34_38-Member Signup and Renewal _ CiviCRM Sandbox on Drupal](https://user-images.githubusercontent.com/6275540/68138833-0d151580-ff21-11e9-8b3e-06532b2b88e9.png)

Reply:
> we need to consider the upgrade impact though as I think this changes previous default behavior?

so I rechecked and 

1- this field is appended dynamically to the webform configuration page (so nothing to worry about regarding upgrading)
2-  by default I set it to "No" which means all memberships are not auto-renew by default either for new or existing webforms which is fine since webform do not have a support for auto-renew without this PR.

So unless I misunderstand what you mean by "changes previous default behavior?" then I think we are good.

Reply:
@omarabuhussein I think it was me that misunderstood the code here :-)  Does this respect the core membership type auto-renew setting? Or can you override here?

Reply:
np :) 

>  Does this respect the core membership type auto-renew setting? Or can you override here?

no it is not, you can override it here.

Reply:
ok